content/webapp/pages/event.tsx

```diff
@@ -144,7 +144,7 @@ function DateList(event: Event) {
 
               {isDayPast(eventTime.range.endDateTime)
                 ? EventStatus({ text: 'Past', color: 'neutral.500' })
-                : eventTime.isFullyBooked
+                : eventTime.isFullyBooked && eventTime.onlineIsFullyBooked
```

Comment:
`eventTime.isFullyBooked` is a bit misleading, can it be changed to `eventTime.isInPersonFullyBooked`

Reply:
Yeah, I'll have a look at renaming in a follow-up PR.